src/js/utils/Badge.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import './utils.scss';
+
+const Badge = ({ label, variant }) => (
+  <span className={`badge ${variant}`}>{label}</span>
```

Comment:
Is this `badge` necessary? You are using, for example, `badge--blue,` so the `badge` seems redundant?